### PR TITLE
47724 test view add product

### PR DESCRIPTION
**Pull request by khieutrung12**
## Related Tickets:

-   [#47724](https://edu-redmine.sun-asterisk.vn/issues/47724)

## WHAT this PR does?

Unit test view add product
User, Brand database seeder

## Notes Impacted Areas:

-   N/A.

## Attachments:

-   N/A.

### CODE REVIEW

**Comment by khieutrung12** [comment]
ready

**Review by DTHNhung — APPROVED** [review]

File: database/seeders/BrandDatabaseSeeder.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     */
+    public function run()
+    {
+        DB::table('brands')->insert([
+            [
+                'name' => 'Akko', 'slug' => 'akko', 'created_at' => new DateTime(), 'updated_at' => new DateTime()
+            ],
+            [
+                'name' => 'Dareu', 'slug' => 'dareu', 'created_at' => new DateTime(), 'updated_at' => new DateTime()
+            ],
```
Comment (ghost) [inline database/seeders/BrandDatabaseSeeder.php]: xuống dòng nhìn cho dễ bn ơi

**Comment by khieutrung12** [comment]
ready

File: database/seeders/BrandDatabaseSeeder.php (hunk 1 above)
Comment (khieutrung12) [reply database/seeders/BrandDatabaseSeeder.php]: @manhtd-1083 vâng ạ

**Pull request merged**